process_report/invoices/billable_invoice.py

```diff
@@ -143,7 +143,10 @@ def get_initial_credit_amount(
 
         current_pi_set = set(data[invoice.PI_FIELD])
         for pi in current_pi_set:
-            pi_projects = data[data[invoice.PI_FIELD] == pi]
+            pi_projects = data[
+                (data[invoice.PI_FIELD] == pi)
+                & ~(data[invoice.SU_TYPE_FIELD].isin(self.EXCLUDE_SU_TYPES))
```

Comment:
You have changed the purpose of this variable, this `pi_projects` doesn't correctly express what this variable does anymore.

Rename it to something like `new_pi_credit_eligible_projects`